### DISCUSSION TITLE
History Log

### DISCUSSION THREAD

**Discussion by JamCam84** [Budibase/budibase#3943] [Help]
I have two tables "Staff" and "Equipment"  and need a log of history when a staff member books out equipment

**Comment by mjashanks** [comment]
You can then add a "Booking Log" table, where `Equipment has many Booking Logs`. This should be possible, let me know if there is anything specific that you would like help with when implementing this.

**Reply by JamCam84** [reply]
If the status (a specific field) changes on the equipment table can I have a form open requesting the relevant info
